src/main/java/com/riskpass/task/Task.java

```
@@ -1,17 +1,22 @@
 package com.riskpass.task;
 
-import com.riskpass.task.config.Config;
+import com.riskpass.task.config.ModifiableConfig;
 import com.riskpass.task.services.ExtractionService;
 
 import java.util.regex.Pattern;
 
 public class Task {
   public static void main(final String[] args) {
-    Config.Modifiable config = new Config.Modifiable();
-    config.commaPattern(Pattern.compile(";"));
-    config.nextQuotePattern(Pattern.compile("\"\\s*;"));
+	ModifiableConfig config = new ModifiableConfig();
+    config.setCommaPattern(Pattern.compile(";"));
+    config.setNextQuotePattern(Pattern.compile("\"\\s*;"));
 
     ExtractionService extractionService = new ExtractionService(config);
-    extractionService.extract(args[0]).forEach(System.out::println);
+    try {
+    	extractionService.extract(args[0]).forEach(System.out::println);
+    } catch (IllegalStateException e) {
```

There are two different opinions on the subject:
1) The Microsoft's one. Hide everything you can hide. User should not see any error even if the error exist. Say them something pleasant like "We are really really sorry but the system is going to die. The error code is 51817, please ask you system administrator, but most like even the god dons't know what to do the next"

2) Open source. Make it cry as loud as possible. Don't hide the errors, display the ugly technical message, this will give you more information, even if your client will be shocked.

The advantage is obvious, when you know that everybody may see you are a stupid idiot, you will do your best to make the code better. So, handle exceptions if
1) you know how to recover
2) you want to add additional (useful) information to the exception

Don't kill the exceptions, do not replace one stupide messages with others.